tests/dummy/app/utils/random.js

```diff
@@ -0,0 +1,25 @@
+var seed = 6;
+// source: http://indiegamr.com/generate-repeatable-random-numbers-in-js/
+// in order to work 'Math.seed' must NOT be undefined,
+// so in any case, you HAVE to provide a Math.seed
+export function randomNumber(max = 1, min = 0, randomSeed = undefined) {
```

Comment:
why add `randomSeed` as the parameter here? I think it should be done separately, as it is currently done, via `setRandomSeed`.